integration-tests/cli/csv-parser-price.it.js

```diff
@@ -119,6 +116,26 @@ describe('CSV and CLI Tests', () => {
         },
       )
     })
+
+    test('should log errors to stderr and log file', (done) => {
```

Comment:
> we decided to log all errors to stderr even if we use stdout 

I see that makes sense as the module has no `continueOnProblems` option. But isn't this what started the issue in the first place?